otelcollector/configmapparser/default-prom-configs/acstorCapacityProvisionerDefaultFile.yml

```diff
@@ -0,0 +1,47 @@
+scrape_configs:
+- job_name: acstor-capacity-provisioner
+  honor_labels: true
+  scrape_interval: $$SCRAPE_INTERVAL$$
+  scheme: http
+  kubernetes_sd_configs:
+    - role: pod
+  relabel_configs:
+    # Include only specified namespace
+  - source_labels: [__meta_kubernetes_namespace]
+    action: keep
+    regex: acstor
+    # Include only specified pods
+  - source_labels: [__meta_kubernetes_pod_label_app_kubernetes_io_name, __meta_kubernetes_pod_label_app_kubernetes_io_component]
+    action: keep
+    regex: capacity-provisioner;capacity-provisioner
+    # Include only specified ports
+  - source_labels: [__meta_kubernetes_pod_container_port_name]
+    action: keep
+    regex: metrics
+
+  # If prometheus.io/path is specified, scrape this path instead of /metrics

```

Comment:
please remove all commented out part of config. I put it for future reference when i converted them...

Reply:
Done